Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM frolvlad/alpine-python3
```

Comment:
@seanh I guess the reason why it still used Python 2 is because the python-pip package depends on Python 2. I can have a look at that so that we use the same base image as with H if that's still a concern?